app/src/main/java/org/listenbrainz/android/ui/screens/login/LoginScreen.kt

```diff
@@ -0,0 +1,206 @@
+        saveUserInfo(userInfo, context)
+    }
+
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+
+        Box() {
+            val comp by rememberLottieComposition(spec = LottieCompositionSpec.RawRes(R.raw.login))
+            LottieAnimation(
+                composition = comp,
+                iterations = LottieConstants.IterateForever,
+                modifier = Modifier.requiredHeightIn(max = 400.dp),
+            )
+        }
+
+        Button(
+            onClick = {
+                when (LBSharedPreferences.loginStatus) {
+                    LBSharedPreferences.STATUS_LOGGED_IN -> logoutUser(context)
+                    else -> startLogin(context)
+
+                }
+            },
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(50.dp)
+                .padding(start = 10.dp, end = 10.dp),
+            colors = ButtonDefaults.buttonColors(backgroundColor = MaterialTheme.colors.onSurface)
+        ) {
+            Text(text = when (LBSharedPreferences.loginStatus) {
+                LBSharedPreferences.STATUS_LOGGED_IN -> stringResource(id = R.string.logout)
+                else -> stringResource(id = R.string.login)
+
+            }, color = Color.White)
+        }
+
+        Text(
+            text = when (LBSharedPreferences.loginStatus) {
+                LBSharedPreferences.STATUS_LOGGED_IN -> stringResource(id = R.string.logout_prompt)
+                else -> stringResource(id = R.string.login_prompt)
+
+            },
+            color = MaterialTheme.colors.surface,
+            fontSize = 18.sp,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.padding(top = 50.dp, start = 5.dp, end = 5.dp)
+        )
+    }
+
+    OnResumeListener{
+        when(it){
+            Lifecycle.Event.ON_RESUME ->{
+                val activity = context.findActivity()
+                val intent = activity?.intent
+                val callbackUri = intent?.data
+                if (callbackUri != null && callbackUri.toString().startsWith(ListenBrainzServiceGenerator.OAUTH_REDIRECT_URI)) {
+                    val code = callbackUri.getQueryParameter("code")
+                    if (code != null) {
+                        loginViewModel.fetchAccessToken(code)
+                    }
+                }
+            }
+            else -> {}
+        }
+    }
+}
+
+
+private fun startLogin(context: Context) {
+    val intent = Intent(
+        Intent.ACTION_VIEW,
+        Uri.parse(
+            ListenBrainzServiceGenerator.AUTH_BASE_URL
+                    + "authorize"
+                    + "?response_type=code"
+                    + "&client_id=" + ListenBrainzServiceGenerator.CLIENT_ID
+                    + "&redirect_uri=" + ListenBrainzServiceGenerator.OAUTH_REDIRECT_URI
+                    + "&scope=profile%20collection%20tag%20rating"
```

Comment:
Let's remove these scopes now? We needed these in the MB android app before but don't see a need here.